File: apiFilRougeIb/Models/Quizz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace apiFilRougeIb.Models
{
    public class Quizz
    {

        public long? IdQuizz { get; set; }
        public string Name { get; set; }
        public long User_idUser { get; set; }
        public long Theme_idTheme { get; set; }
        public string Code { get; set; }
        public string? Comment { get; set; }
        public DateTime? DateFermeture { get; set; }
        public int? Timer { get; set; }
        public int Level_idLevel { get; set; }

        public Quizz() { }
        public Quizz(string name, long user_idUser, long Theme_idTheme,string code,
            DateTime? dateFermeture, int level_idlevel, string? comment, int? timer, 
            long? idQuizz = null)
        {
            this.Name = name;
            this.User_idUser = user_idUser;
            this.IdQuizz = idQuizz;
            this.Theme_idTheme = Theme_idTheme;
            this.Code =code;
            this.DateFermeture =dateFermeture;
            this.Timer =timer;
            this.Comment = comment;
            this.Level_idLevel = level_idlevel;

        }
    }
}

File: angularFilRougeIb/src/app/_models/quizz.ts
export class Quizz{

    private  _idQuizz :number;  
    private  _name :string; 
    private _user_idUser  :number;
    private  _theme_idTheme  :number;
    private  _code  :string;
    private  _dateClosed :Date;
    private  _timer  :number;
    private _level_idLevel  :number;

    get idQuizz() : number {
        return this._idQuizz;
      }
    
    get name() : string {
    return this._name;
    }

    set name(Name : string){
        this._name = Name;
    }

    get  user_idUser() : number {
        return this._user_idUser;
        }

    set  user_idUser( User_idUser : number){
        this._user_idUser =  User_idUser;
    }
    get  theme_idTheme() : number {
        return this._theme_idTheme;
        }

    set  theme_idTheme( Theme_idTheme : number){
        this._theme_idTheme =  Theme_idTheme;
    }

    get code() : string {
        return this._code;
        }
    
    set code(Code : string){
        this._code = Code;
    }

    get dateClosed() : Date {
        return this._dateClosed;
        }
    
    set dateClosed(DateClosed : Date){
        this._dateClosed = DateClosed;
    }

    get timer() : number {
        return this._timer;
        }
    
    set timer(timer : number){
        this._timer = timer;
    }

    get level_idLevel() : number {
        return this._level_idLevel;
        }
    
    set level_idLevel(Level_idLevel : number){
        this._level_idLevel = Level_idLevel;
    }
}